### PR TITLE
Fixing SLES12 issue

### PR DESCRIPTION
**Pull request by Evalle**
Signed-off-by: evalle shmarnev@gmail.com

### CODE REVIEW

**Comment by cpuguy83** [comment]
ping @tianon

**Comment by estesp** [comment]
I didn't know if the `[d,s]` construct relied on a specific bash-ism (given this script only requires `/bin/sh` we can't rely on that being bash).. but via local testing I see that case construct works in bash, dash, and busybox sh, so looks like that's safe.  @tianon can confirm/deny I'm sure :)

**LGTM**

**Comment by rtrauntvein** [comment]
LGTM +1

**Comment by Evalle** [comment]
ping  @tianon

File: hack/install.sh [hunk 1]
```diff
@@ -140,7 +140,7 @@ do_install() {
 			exit 0
 			;;
 
-		'opensuse project'|opensuse|'suse linux'|sled)
+		'opensuse project'|opensuse|'suse linux'|sle[d,s])
```
Comment (tianon) [inline hack/install.sh]: The comma here is superfluous if the goal is just to match both `sles` and `sled` (as-is, this will also match `sle,`). :+1:

**Comment by Evalle** [comment]
@jfrazelle could you please review this ?

**Comment by jfrazelle** [comment]
LGTM, i would just fix what tianon pointed out

**Comment by Evalle** [comment]
Thank you, @jfrazelle @tianon  I will delete this comma.

**Comment by Evalle** [comment]
@jfrazelle 
I can't modify current PR because I lost all my data with SDD failure :\ So I created a new PR (https://github.com/docker/docker/pull/14893) 
I will close current PR.

**Pull request closed without merging**